src/uct/api/uct.h

```diff
@@ -2562,6 +2562,16 @@ ucs_status_t uct_md_mem_advise(uct_md_h md, uct_mem_h memh, void *addr,
                                size_t length, uct_mem_advice_t advice);
 
 
+/**
+ * @ingroup UCT_MD
+ * @brief Register global VA for zero-copy sends and remote access.
+ *
+ * @param [in]     md        Memory domain to register memory on.
+ * @param [out]    memh_p    Filled with handle for create object.
+ */
+ucs_status_t uct_md_mem_reg_gva(uct_md_h md, uct_mem_h *memh_p);
```

Comment:
1. move it to uct_v2.h
2. add (empty) params struct
3. maybe extend uct_md_mem_reg_v2() instead? for example, add a flag